pootle/core/views.py

```diff
@@ -464,7 +465,8 @@ def is_admin(self):
 
     @property
     def language(self):
-        return None
+        if self.tp:
+            return self.tp.language
```

Comment:
it looks implicit. can we comment which views are going to use this (not overwritten) method?